ts/features/itwallet/common/utils/itwClaimsUtils.ts

```diff
@@ -227,9 +227,22 @@ export type DrivingPrivilegesClaimType = t.TypeOf<
 export const FiscalCodeClaim = PatternString(FISCAL_CODE_WITH_PREFIX);
 
 /**
- * Alias for the string fallback of the claim field of the credential.
+ * Empty string fallback of the claim field of the credential.
  */
-export const PlainTextClaim = t.string;
+export const EmptyStringClaim = new t.Type<string, string, unknown>(
+  "EmptyString",
+  (input: unknown): input is string => input === "", // Type guard
+  (input, context) =>
+    typeof input === "string" && input === ""
+      ? t.success(input)
+      : t.failure(input, context, "Expected an empty string"),
+  t.identity
+);
```

Comment:
Did you try with a `PatternString("^$")`. It works on strings and it matches only "". 
What do you think?

Reply:
Doing so creates an overlap in types which makes the claim in the last case, `UnknownClaimItem`,  to be wrongly typed as `const decoded: "******"`. Do we want to lose the exhaustive type checking in this case? 

This also made me notice that we hiding them, the empty claim would reappear. I fixed it in ead1721711efa50ce0413f63f45ee0d384b1799c

Reply:
Well, `UnknownClaimItem` should be the fallback for claims we can't/won't map to a component. I don't see any problem to not use an exhaustive type checking. 
Mine was just a proposal to keep the approach as simple as possible.